doc/user/content/sql/functions/date-part.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "date_part Function"
+description: "Returns a specified time component from a time-based value"
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`date_part` is modeled on the traditional Ingres equivalent to the SQL-standard 
```

Comment:
Can this doc string be changed to summarize what the function does? Mentioning Ingres (I have no idea what that is myself), the sql spec, or postgres compat are not useful if a reader doesn't know what those are. If you think that's important information to discuss, it should be below the return values section. Other functions have a Details section for discussing this kind of stuff.

Reply:
Followup: I see that this is exactly how postgres documents date_part and that is maybe useful. So I guess ignore my comment and this can stay like it is?

Reply:
Yeah I tried to model these docs somewhat after the equivalent postgres docs found here: https://www.postgresql.org/docs/14/functions-datetime.html#FUNCTIONS-DATETIME-EXTRACT.

Ingres (https://dbdb.io/db/ingres) is an older database that Postgres is was modeled on (Postgres -> Post Ingres), and it seems like the only reason Postgres even has the `date_part` function was for compatibility with Ingres. Though after Postgres chaged `EXTRACT` to return numeric, they mention in the discussion and PR that `date_part` can be used as a faster less precise version of `EXTRACT`.

I can change the beginning to 
> `date_part` returns some time component from a time-based value, such as the year from a Timestamp. It is modeled on the traditional Ingres...

I'm also happy to remove the Ingres and SQL-standard mentions if we don't care about them.

Tagging @sploiselle since there was another comment about Ingres elsewhere.

Reply:
I think removing the mentions is good, unless you can point someone to a doc/reference that will enhance their understanding of the API.

Reply:
Agreed. I can’t imagine that in 2022 that comparison is useful to anyone.  I’ll bet $20 that that line in the PostgreSQL document dates back to pre 2000.

Reply:
@#$%@ I was off by one year: github.com/postgres/postgres/commit/e6767a32053963a968f419256e3629ace5e8df2f

Reply:
So I removed the mentions to Ingres and the SQL standard. Let me know what you think. I'm honestly not great at technical writing so feel free to make suggestions to grammar and content.